python/tank/util/yaml_cache.py

```diff
@@ -0,0 +1,138 @@
+# Copyright (c) 2015 Shotgun Software Inc.
```

Comment:
No worries and thanks for thinking about it ;) - I did originally consider changing the yaml library to do the caching but I didn't want to change what is essentially a 3rd party library so instead this is intended as a very simple wrapper interface.